src/mxarray.jl

```diff
@@ -234,15 +235,15 @@ function size(mx::MxArray, d::Integer)
     if d <= 0
         throw(ArgumentError("The dimension must be a positive integer."))
     end
-    
```

Comment:
Wasn't the conclusion that the `transcode` should stay? Even if it doesn't the change should not be in this PR.